chain/vm/vmi.go

```diff
@@ -25,11 +25,17 @@ var useFvmForMainnetV15 = os.Getenv("LOTUS_USE_FVM_TO_SYNC_MAINNET_V15") == "1"
 
 func NewVM(ctx context.Context, opts *VMOpts) (Interface, error) {
 	if opts.NetworkVersion >= network.Version16 {
+		if os.Getenv("LOTUS_FVM_DEBUG") == "1" {
```

Comment:
Put this in a global var so we only check it once?

Reply:
sure